src/Command/RotateCloudSqlPasswordCommand.php

```diff
@@ -0,0 +1,270 @@
+<?php
+
+namespace App\Command;
+
+use Exception;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+use Symfony\Component\Process\Process;
+
+class RotateCloudSqlPasswordCommand extends Command
+{
+    private const PASSWORD_LENGTH = 24;
+    protected static $defaultName = 'app:rotate-cloudsql-password';
+
+    private SymfonyStyle $io;
+    private ParameterBagInterface $params;
+
+    public function __construct(ParameterBagInterface $params)
+    {
+        $this->params = $params;
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Rotates MySQL user passwords for a Cloud SQL instance and updates Secret Manager.')
+            ->addArgument('instance', InputArgument::REQUIRED, 'The Cloud SQL instance name.')
+            ->addOption('project', null, InputOption::VALUE_OPTIONAL, 'The Google Cloud project ID. If not provided, the instance name is used.')
+            ->addOption('dry-run', null, InputOption::VALUE_NONE, 'Perform a dry run without making any changes.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->io = new SymfonyStyle($input, $output);
+        $instance = $input->getArgument('instance');
+        $project = $input->getOption('project') ?? $instance;
+        $dryRun = $input->getOption('dry-run');
+
+        if ($dryRun) {
+            $this->io->note('-- DRY RUN MODE --');
+        }
+
+        if (!$this->io->confirm("Are you sure you want to rotate passwords for instance \"{$instance}\" in project \"{$project}\"?", false)) {
+            $this->io->warning('Command aborted.');
+            return Command::SUCCESS;
+        }
+
+        try {
+            $users = $this->params->get('cloudsql_users');
```

Comment:
Where do the users come from is this in a config file somewhere or an environment variable?

Reply:
Yes, users are defined in config.yml file, I just added an example in the dist file